.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,2 @@
+# This module contains no acceptance tests - this is a placeholder for
+# modulesync.
```

Comment:
With this being an empty placeholder for now, disabling Travis means we would have no acceptance tests in CI.

So, the plan would be that we are running these locally only during the meantime?

Reply:
good catch, this was a bad sync on my part and thus it got rendered incorrectly

Reply:
Fixed now